src/app/components/SendForm/ChainSend.tsx

```diff
@@ -204,7 +202,7 @@ class ChainSend extends React.Component<Props, State> {
 }
 
 export default connect<StateProps, DispatchProps, OwnProps, AppState>(
-  state => ({
+  (state) => ({
```

Comment:
Can we enable https://prettier.io/docs/en/options.html#arrow-function-parentheses to reduce the size of the diff with the prettier upgrade?

Reply:
:+1: 
done in https://github.com/joule-labs/joule-extension/pull/257/commits/2998810ead18c51e105a306798e1c02e25453e56 
configuration changed in https://github.com/joule-labs/joule-extension/pull/257/commits/f39e9fd5f2fbf730d221a1b575a1d3ac9a4210b3